fcm-channel/src/main/java/io/fcmchannel/sdk/chat/ChatMessageViewHolder.java

```diff
@@ -104,7 +121,6 @@ void bindView(Message chatMessage) {
         this.chatMessage = chatMessage;
 
         message.setText(chatMessage.getText());
-        Linkify.addLinks(message, Linkify.ALL);
```

Comment:
The links sent in the chat are still clickable without this line?

Reply:
I put the attribute `android:autoLink="web"` in the message text view.